utils/DateUtils.js

```diff
@@ -39,6 +39,11 @@ class DateUtils {
     return moment().subtract(1, 'month');
   }
 
+  static getLast30days(language) {
```

Comment:
Ambiguous method name. does this return a random date in the last 30 days?

Reply:
Yes, an helper method to generates a random date between 1 - 30 because submitting an SYA case with same MRN date throws duplication error

Reply:
can the method name be more specific. e.g. `getRandomDateInLast30Days`